### PR TITLE
Adds my blog post about team communication

### PR DESCRIPTION
**Pull request by rgbcolor**

### CODE REVIEW

File: source/posts/2016-05-25-how-does-project-communication-become-complex.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+social: true
+author: Maria Matveeva
+twitter: "rgbcolor"
+summary: "Learning about communication patterns in larger teams"
+published: true
+tags: design, business, observations, product management, team
+---
+
+
+Projects of larger scale or longer duration often involves a large design team. We typically keep everyone informed and share information efficiently on teams roughly the size of a large family. But we don't have that intuitive understanding of how teams work once they get much larger than that.
```
Comment (rsocci) [inline source/posts/2016-05-25-how-does-project-communication-become-complex.md]: `involves` -> `involve`

File: source/posts/2016-05-25-how-does-project-communication-become-complex.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Manage the conversations on larger teams. Because you have to.
+
+![layered communication](https://i.imgur.com/emvoO7J.jpg)
+
+The answer to the math problem is structure. As team sizes grow, managers or team leaders emerge who become responsible for keeping on top of communications within a smaller team (think - the drawing of four people above) as well as for relaying that information to other team leaders. 
+
+## Reinventing the wheel?
+
+The need for management is not a new concept. As I was discovering it for myself, I was surely reinventing the wheel. Colleagues advised that the number of connections between nodes is described by [https://en.wikipedia.org/wiki/Metcalfe%27s_law](Metclafe’s Law).
```
Comment (rsocci) [inline source/posts/2016-05-25-how-does-project-communication-become-complex.md]: The content inside the brackets and parentheses for your link should be swapped: `[Metclafe's Law](url)`

File: source/posts/2016-05-25-how-does-project-communication-become-complex.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  
+![Metclafe’s Law](https://i.imgur.com/Iv59ich.png)
+
+
+
+
+
+The information exchange in a group could be managed by breaking down & [https://en.wikipedia.org/wiki/Stakeholder_management](managing different stakeholders) depending on their influence (power) and influence level.
```
Comment (rsocci) [inline source/posts/2016-05-25-how-does-project-communication-become-complex.md]: same thing with the link here

File: source/posts/2016-05-25-how-does-project-communication-become-complex.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+The information exchange in a group could be managed by breaking down & [https://en.wikipedia.org/wiki/Stakeholder_management](managing different stakeholders) depending on their influence (power) and influence level. 
+
+
+
+![Stakeholder Management](https://i.imgur.com/MpNKIY0.png) 
+
+
+
+I also learned that [https://blog.bufferapp.com/small-teams-why-startups-often-win-against-google-and-facebook-the-science-behind-why-smaller-teams-get-more-done](coordination cost) becomes one of the factors in determining the productivity of a team.
```
Comment (rsocci) [inline source/posts/2016-05-25-how-does-project-communication-become-complex.md]: same thing with the link here

**Comment by rsocci** [comment]
Nice post @rgbcolor!

File: source/posts/2016-05-25-how-does-project-communication-become-complex.md (hunk 4 above)
Comment (rgbcolor) [inline source/posts/2016-05-25-how-does-project-communication-become-complex.md]: Thank you @rsocci - you caught all the things!

**Pull request merged**